src/main/java/core/basesyntax/service/impl/FruitStorageServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitStorageService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitStorageServiceImpl implements FruitStorageService {
+    private FruitStorageDao fruitStorageDao;
+    private OperationStrategy operationStrategy;
+
+    public FruitStorageServiceImpl(FruitStorageDao fruitStorageDao,
+                                   OperationStrategy operationStrategy) {
+        this.fruitStorageDao = fruitStorageDao;
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public void processTransactions(List<FruitTransaction> transactions) {
+        for (FruitTransaction fruitTransaction : transactions) {
+            operationStrategy.get(fruitTransaction.getOperation())
+                    .updateFruitStorage(fruitTransaction, fruitStorageDao);
+        }
+        System.out.println("All transactions processed successfully");
+    }
```

Comment:
It would be better practice not to catch or handle exceptions silently within service methods. If an exception occurs during `operationStrategy.get(...)`, it should be propagated up to the caller of `processTransactions` so that it can be handled appropriately there.